renovate.json

```diff
@@ -15,6 +16,11 @@
     },
     {
       "matchPackageNames": ["github.com/openstack-k8s-operators"],
+      "schedule": [
+        "after 10pm every weekday",
+        "before 3am every weekday",
```

Comment:
that is 9:00 am in CET which looks good to me, I can start my day to look at the test result of the PR proposed by the bot.